umbraco/Umbraco-CMS
# v8 - Convert relation type editor to Angular

### Prerequisites


If there's an existing issue for this PR then this fixes: #3436 

### Description
Converts the legacy web forms-based relation types editor to Angular, and removes the legacy views.

#### Test steps
1. Open an existing relation type and verify that the data is displayed correctly
2. Create new relations associated with a relation type and verify that they appear in the relations table. I quickly tested this by copying pages with the "relate to original" option selected.
3. Create a new relation type and verify that the data is displayed correctly
4. Delete a relation type and verify that it has been removed from the database.

#### Notes
* The Refresh action when right clicking on the root folder doesn't work, although this seems to be a v8 issue.
* When renaming a relation type the alias is also updated. I couldn't see how to lock this down so that the user has to unlock the alias before it can be edited.
